pages/metadata.md

```diff
@@ -0,0 +1,169 @@
+good excerpt is usually no more than two sentences long. No Markdown
+allowed.
+
+Description
+-----------
+
+A short, plain-text description of your post, surrounded with quotation
+marks. No Markdown or HTML allowed. This can be the same as the excerpt
+or it can be a sentence or two that you feel represents your post well.
+It may appear next to your post on social media and other places that
+fetch article metadata.
+
+And here are the optional fields:
+
+Image
+-----
+
+The main image of your post. A relative link, with a leading /. This
+will appear in social media platforms when the post is shared. It can be
+a different image than those that appear embedded in your post. Though
+this section is optional, we highly encourage you to include a relevant
+picture. It will increase the likelihood of your post being shared on
+social media and can help give readers more information. Authors do not have to fill in the image path. You can simply insert your image into your Google Doc draft and a member of the blog team will fill it in during publishing. Full-width images should be 1600px wide. If omitted, a
+standard image is used.
+
+Date
+-----
+
+Overrides the default date set by Jekyll. When you create a post, you
+give it a filename with a date in it. For example:
+2015-10-01-post-title.md will automatically use October 1 2015 for the
+date and /2015/10/01/post-title/ for the URL. If you don’t want the
+publish date to match the filename, you can set this with a date string
+in the form of YYYY-MM-DD. For example, date: 2015-01-01 would set the
+date to January 1 2015. Published posts with a date in the future will
+not appear on the site. Posts published on the same day will be listed
+in chronological order instead of by most recent. To have posts listed
+with the most recent appearing first, you can add a timestamp to the
+front matter in the form YYYY-MM-DD HH:MM using 24-hour time.
+
+Example front matter
+--------------------
+```
+---
+title: "New Federalist platform lets agencies quickly launch websites"
+date: 2015-09-15
+authors:
+- melody
+tags:
+- our projects
+- platform
+- proactive federal partner
+excerpt: "18F’s new Federalist platform is a suite of tools designed to
+make it faster for government agencies to build websites that are
+secure, responsive, and accessible."
+description: "18F’s new Federalist platform is a suite of tools designed
+to make it faster for government agencies to build websites that are
+secure, responsive, and accessible."
+image: /assets/blog/federalist/sbst-screenshot.jpg
+---
+```
+
+Here’s a template you can paste directly into your draft:
+
+```
+---
+title: ""
+date: YYYY-MM-DD (optional)
+authors:
+-
+tags:
+-
+-
+-
+excerpt: ""
+description: ""
+image: /assets/blog/slug/image.jpg/png/gif
+---
+```
+
+Testing
+-------
+
+Every time we publish a blog post, GitHub runs a suite of tests that
+check that the front matter for each post is correct. Right now,
```

Comment:
+1 on adding the link. Do you want to include the link thing? This section is just about front matter/metadata so it might be a bit out of place. Maybe just trust that if people want to know more about the tests they will read the post?